### DISCUSSION TITLE
Compile?

### DISCUSSION THREAD

**Discussion by Sheeeeit** [pierrelasse/YimStuff#12] [Q&A]
Hello! Was wondering if I can compile the scripts myself? In that case, what do I need to do?

**Comment by pierrelasse** [comment]
Just type `t update` in the console.
Make sure that your CWD is in the project directory.
It should automaticly copy the script into your yimmenu scripts folder.

**Reply by Sheeeeit** [reply]
I have Windows and when try execute `t.bat` just got `<help>`

`PS C:\>  & 'C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python39_64\python.exe' 'c:\Users\Cat\.vscode\extensions\ms-python.python-2023.22.1\pythonFiles\lib\python\debugpy\adapter/../..\debugpy\launcher' '61230' '--' 'c:\Users\Cat\Desktop\YimStuff-master\t\main.py'
<help>
PS C:\>`

And no more happened... Can would be problem in my OS?

**Reply by pierrelasse** [reply]
Did you type anything after it?

**Reply by Sheeeeit** [reply]
No

**Reply by pierrelasse** [reply]
Then try typing `update` after it

**Reply by Sheeeeit** [reply]
Output file will called `SussySpt.lua`?

**Reply by pierrelasse** [reply]
If you use update, you just have to reload your scripts in yimmenu.

**Reply by Sheeeeit** [reply]
After `t update` i got two files in `%appdata%/roaming/yimmenu/scripts`;
- `SussySpt.lua`
- `yimutils.lua`

It's good or i should got only one `SussySpt.Merged.lua`?

**Reply by pierrelasse** [reply]
Yes, now you just have to reload your scripts in yimmenu and you're done.

**Reply by Sheeeeit** [reply]
Done. It compiled. Thank you for help! :)

**Reply by pierrelasse** [reply]
Np
